data/hai missions.txt

```diff
@@ -1061,3 +1061,366 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+mission "Hai Solifuge response"
+	landing
+	invisible
+	to offer
+		has "Wanderers Hai Assistance 3: done"
+	on offer
+		event "Hai Pond Striders released" 10
+		fail
+
+event "Hai Pond Striders released"
+	fleet "Large Hai"
+		add variant 3
+			"Hai Pond Strider" 2
+			"Flea" 16
+			"Lightning Bug" 2
+		add variant 2
+			"Hai Pond Strider" 3
+			"Flea" 24
+		add variant 1
+			"Hai Pond Strider"
+			"Flea" 8
+			"Geocoris"
+			"Aphid (Anti-Missile)" 2
+			"Grasshopper"
+		add variant 1
+			"Hai Pond Strider" 2
+			"Flea" 16
+			"Centipede"
+
+mission "Strider: View of the Pond"
+	description "Deliver a message from Ambassador Sayari to Hai Home."
+	name "Message Hai-home"
+	landing
+	source "Vara K'chrai"
+	destination "Hai-home"
+	to offer
+		has "Wanderers Solifuge Recon 1: done"
+	on offer
+		conversation
+			`"Ambassador Sayari has been informed of the Solifuge, has a message for the Hai Homeworld, and needs your help to deliver it to Hai-Home."`
+				accept
+	on complete
+		conversation
+			`The Hai, as usual, mysteriously and efficiently deliver your message from Ambassador Sayari, and a local prefect appears within minutes to inquire if you've encountered a friendly race with hull repair technology.`
```

Comment:
in all honesty, i'm not a story writer, and i've been disconnected from ES for so long, this mission string really feels a bit like talking to the Quarg/Wanderers/Remnants all over - because it's really just a base plot with the hopes that a bonafide writer will pick up the scraps and make a nice read out of it...

i've done some touchup in the upcoming commit...